WordPress/src/main/java/org/wordpress/android/ui/main/WPMainActivity.java

```diff
@@ -71,6 +65,8 @@
 import org.wordpress.android.fluxc.store.SiteStore.OnSiteChanged;
 import org.wordpress.android.fluxc.store.SiteStore.OnSiteEditorsChanged;
 import org.wordpress.android.fluxc.store.SiteStore.OnSiteRemoved;
+import org.wordpress.android.inappupdate.IInAppUpdateManager;
```

Comment:
Small nit: I know this is not a change you made, but is there a typo in the naming of `IInAppUpdateManager`?

Reply:
I am not sure. @pantstamp, is there a typo in `IInAppUpdateManager`, or is it Interface's 'I' that we are used to seeing in C# code style?